crates/fluvio-cluster/src/progress.rs

```diff
@@ -6,6 +6,8 @@ use crate::render::{ProgressRenderedText, ProgressRenderer};
 
 #[derive(Debug)]
 pub(crate) enum InstallProgressMessage {
+    LaunchingK8s,
```

Comment:
Should hide internal embedded.  Maybe `Metadata`?

Reply:
Done